src/Frontend/Modules/Blog/Engine/Model.php

```diff
@@ -874,29 +867,25 @@ public static function notifyAdmin(array $comment): void
             $from = FrontendModel::get('fork.settings')->get('Core', 'mailer_from');
             $replyTo = FrontendModel::get('fork.settings')->get('Core', 'mailer_reply_to');
             $message = Message::newInstance(FL::msg('NotificationSubject'))
-                ->setFrom([$from['email'] => $from['name']])
-                ->setTo([$to['email'] => $to['name']])
-                ->setReplyTo([$replyTo['email'] => $replyTo['name']])
-                ->parseHtml(
-                    '/Core/Layout/Templates/Mails/Notification.html.twig',
-                    $variables,
-                    true
-                )
-            ;
+                              ->setFrom([$from['email'] => $from['name']])
+                              ->setTo([$to['email'] => $to['name']])
+                              ->setReplyTo([$replyTo['email'] => $replyTo['name']])
+                              ->parseHtml(
+                                  '/Core/Layout/Templates/Mails/Notification.html.twig',
+                                  $variables,
+                                  true
+                              );
             FrontendModel::get('mailer')->send($message);
         }
     }
 
     /**
      * Parse the search results for this module
-     *
```

Comment:
did you do this on purpose?

because the empty line needs to be there according to the spec

Reply:
Fixed in ae614a62d315d074c1532c1f207abb2bd14d4a99